source/tutorials/fmv.rst

```diff
@@ -1,36 +1,46 @@
 .. _fmv:
 
-Function multi-versioning
+Function Multi-Versioning
 #########################
 
+In this tutorial, we will use :abbr:`FMV (Function Multi-Versioning)` on
+general code and on :abbr:`FFT (Fast Fourier Transform)` library code (FFTW).
+Upon completing the tutorial, you will be able to use this technology on your
+code and use the libraries to deploy architecture-based optimizations to your
+application code.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Description
+***********
+
 CPU architectures often gain interesting new instructions as they evolve but
 application developers find it difficult to take advantage of those
 instructions. The reluctance to lose backward-compatibility is one of the
 main roadblocks slowing developers from using advancements in newer computing
-architectures. :abbr:`FMV (Function Multi-Versioning)`, which first appeared
-in `GCC`_ 4.8, is a way to have multiple implementations of a function, each
-using a different architecture specialized instruction-set extensions. GCC
-6 introduces changes to FMV to make it even easier to bring architecture-
-based optimizations to the application code.
+architectures. FMV, which first appeared in `GCC`_ 4.8, is a way to have
+multiple implementations of a function, each using different
+architecture-specialized instruction-set extensions. GCC 6 introduces
+changes to FMV to make it even easier to bring architecture-based
+optimizations to the application code.
 
-In this tutorial we will use FMV on general code and on
-:abbr:`FFT (Fast Fourier Transform)` library code (FFTW). Upon completing the
-tutorial, you will be able to use this technology on your code and use the
-libraries to deploy architecture-based optimizations to your application code.
 
 Install and configure a |CL| host on bare metal
 ***********************************************
+
 First, follow our guide to :ref:`bare-metal-install-desktop`. Once the bare
 metal installation and initial configuration are complete, add the
-`desktop-dev` bundle to the system. `desktop-dev`: contains the necessary
-development tools like GCC and Perl\*.
+:command:`desktop-dev` bundle to the system. :command:`desktop-dev` contains
+the necessary development tools like GCC and Perl\*.
 
-To install the bundles, run the following command in the :file:`$HOME`
-directory:
+#.  To install the bundles, run the following command in the :file:`$HOME`
+    directory:
 
-.. code-block:: bash
+  .. code-block:: bash
 
-   sudo swupd bundle-add desktop-dev
+     sudo swupd bundle-add desktop-dev
```

Comment:
Please double-check to make sure `sudo` is left-aligned with `code`, above. Thank you.

Reply:
Done.